tests/unit/jobs/ContentJobsTest.php

```diff
@@ -653,6 +653,24 @@ public function canRestoreContent()
         $this->assertNull($content->deleted_at);
     }
 
+    /** @test */
+    public function canRestoreContentWithChildren()
```

Comment:
I'd rename this function to something like:
- can restore content **without** children
or
- restores content **without** children